tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
     pytest
     pytest-cov
     git+https://github.com/packit-service/requre
+    Deprecated
```

Comment:
Here, we need to specify test-only packages. => Package `Deprecated` should be installed with the package itself.
(Try removing the tox cache directory if you have some problems with that.)